src/main/java/com/tstepnik/planner/domain/Statistics.java

```diff
@@ -0,0 +1,72 @@
+package com.tstepnik.planner.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.net.Inet4Address;
+
+@Entity
+public class Statistics {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Long userId;
+
+    private Double productivity;
+
+    private Integer numberOfTasks;
```

Comment:
Is this number of total archived tasks?

Reply:
Total archived user tasks.

Reply:
I think we could use name change so it will be more descriptive.

Reply:
bump